grc/gui/canvas/param.py

```diff
@@ -86,33 +86,36 @@ def format_tooltip_text(self):
             tooltip_lines.extend(' * ' + msg for msg in errors)
         return '\n'.join(tooltip_lines)
 
+
+
+    ##################################################
+    # Truncate helper method
+    ##################################################
+    def truncate(self, string, style=0):
```

Comment:
You could have made this static, but that's a teeny tiny nit.

Reply:
I didn't think it could be since it uses "self.name".  I think if you want this to be static you would have to pass the name (or, alternatively, max_len) in from the caller right?